<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qf::qmlwidgets::reports::TableViewChooseColumnsWidget</class>
 <widget class="QWidget" name="qf::qmlwidgets::reports::TableViewChooseColumnsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grpColumns_2">
     <property name="title">
      <string>exported columns</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QToolButton" name="btColumnShiftUp">
          <property name="toolTip">
           <string>shift selected column up</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btColumnShiftDown">
          <property name="toolTip">
           <string>shift selected column down</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btColumnsRemoveSelected">
          <property name="toolTip">
           <string>Remove selected columns</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../qfqmlwidgets.qrc">
            <normaloff>:/qf/qmlwidgets/images/minus.png</normaloff>:/qf/qmlwidgets/images/minus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tblExportedColumns">
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpColumns">
     <property name="title">
      <string>source columns</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QToolButton" name="btColumnsAll">
          <property name="toolTip">
           <string>select all</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/libqfgui/images/ok.png</normaloff>:/libqfgui/images/ok.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btColumnsNone">
          <property name="toolTip">
           <string>select none</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/libqfgui/images/close.png</normaloff>:/libqfgui/images/close.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btColumnsInvert">
          <property name="toolTip">
           <string>invert selection</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../qfqmlwidgets.qrc">
            <normaloff>:/qf/qmlwidgets/images/reload.png</normaloff>:/qf/qmlwidgets/images/reload.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btColumnsAppendSelected">
          <property name="toolTip">
           <string>invert selection</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../qfqmlwidgets.qrc">
            <normaloff>:/qf/qmlwidgets/images/plus.png</normaloff>:/qf/qmlwidgets/images/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="columnsTabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabModel">
         <attribute name="title">
          <string>model</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="lstModelColumns">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>table</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListWidget" name="lstTableColumns">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qfqmlwidgets.qrc"/>
 </resources>
 <connections/>
</ui>
